circle/functions

```diff
@@ -427,3 +427,38 @@ chart_update_version() {
     git commit -m "$CHART_NAME: bump chart version to \`$CHART_VERSION_NEXT\`" >/dev/null
   fi
 }
+
+get_image_build_dir() {
+    local RELEASE_SERIE=${1}
+    local BASE_IMAGE=${2}
+
+    if [[ -f $RS/Dockerfile ]]; then
```

Comment:
use the local variables `RELEASE_SERIE` and `BASE_IMAGE` in the function context (do not reference `BI`, `RS`, etc.) and also use `$DOCKERFILE` in place of `Dockerfile`. We allow specifying a different Dockerfile name via this variable. Used in che branches where the file is name `.codenvy.dockerfile.`

eg. `if [[ -f $RELEASE_SERIE/$DOCKERFILE ]]` and so on..